### ISSUE TITLE
Upgrade to BootstrapV4

### ISSUE THREAD

**Issue by amoraitis** [amoraitis/TodoList#13]
Let´s upgrade the project´s bootstrap to 4 and make the required changes to the website to be functional!

The process is written [here](http://amoraitis.me/net-core/2018/11/05/bootstrap4-in-aspnet-core-mvc-identity.html), step-by-step.

**Comment by mariombaltazar** [comment]
Hi,
Can i contribute with this issue?

**Comment by amoraitis** [comment]
@mariombaltazar Of course, follow my article [here](http://amoraitis.me/net-core/2018/11/05/bootstrap4-in-aspnet-core-mvc-identity.html). I am waiting for your PR

**Comment by mariombaltazar** [comment]
> @mariombaltazar Of course, follow my article [here](http://amoraitis.me/net-core/2018/11/05/bootstrap4-in-aspnet-core-mvc-identity.html). I am waiting for your PR

Thanks @amoraitis

**Comment by pirland** [comment]
I'll take a wack at this, if nobody else is working on it.

**Comment by amoraitis** [comment]
@pirland keep in mind that current version of bootstrap is 4,we just want to update obsolete references to css classes. I have mentioned some in the article.
Tahnks

**Comment by pirland** [comment]
This should be taken care of.  https://github.com/amoraitis/TodoList/pull/51

**Comment by bsimser** [comment]
What's outstanding on this? #51 was merged in so what's left to do to close this?
